plenum/test/txn_author_agreement/conftest.py

```diff
@@ -73,3 +76,31 @@ def taa_expected_data(taa_input_data):
 def taa_expected_digests(taa_input_data):
     # TODO use some other API, e.g. sdk's one
     return {data.version: ConfigReqHandler._taa_digest(data.text, data.version) for data in taa_input_data}
+
+
+@pytest.fixture(scope="function")
+def taa_aml_request(sdk_wallet_trustee):
+    return Request(identifier=sdk_wallet_trustee[1],
+                   reqId=5,
+                   protocolVersion=CURRENT_PROTOCOL_VERSION,
+                   operation={'type': TXN_AUTHOR_AGREEMENT_AML,
+                              AML_VERSION: randomString(),
+                              AML: {'Nice way': 'very good way to accept agreement'},
+                              AML_CONTEXT: randomString()})
+
+
+@pytest.fixture(scope="module")
+def taa_aml_request_module(sdk_wallet_trustee):
+    return Request(identifier=sdk_wallet_trustee[1],
+                   reqId=5,
+                   protocolVersion=CURRENT_PROTOCOL_VERSION,
+                   operation={'type': TXN_AUTHOR_AGREEMENT_AML,
+                              AML_VERSION: randomString(),
+                              AML: {'Nice way': 'very good way to accept agreement'},
+                              AML_CONTEXT: randomString()})
+
+
+@pytest.fixture(scope="module")
+def setup(looper, txnPoolNodeSet, taa_aml_request_module, sdk_pool_handle, sdk_wallet_trustee):
```

Comment:
Probably `nodeSetWithTaaAml` would be a better name for this?

Reply:
changed to setup_aml